src/main/java/org/xstefank/check/TemplateChecker.java

```diff
@@ -23,21 +23,21 @@ public TemplateChecker(FormatConfig config) {
     }
 
     public String checkPR(JsonNode payload) {
-        log.info("checking PR");
-        String description = "";
+        log.debug("checking PR\n" + payload);
```

Comment:
@lukasburda you've filed windows issue lately. I believe this end of line is not going to work there